packages/@ourworldindata/grapher/src/modal/SourcesModal.tsx

```diff
@@ -91,8 +91,8 @@ export class SourcesModal extends React.Component<
 
     @computed private get showStickyModalHeader(): boolean {
         const modalWidth = this.modalBounds.width - 2 * this.modalPadding
-        const dismissButtonWidth = 32
-        return (modalWidth - MAX_CONTENT_WIDTH) / 2 < dismissButtonWidth
+        const closeButtonWidth = 32
```

Comment:
Maybe it makes sense to make this a constant inside `CloseButton.tsx` that we can import here, so it's easier to keep in sync and find it.

Reply:
Good call!